schemas/Clinical/index.js

```diff
@@ -354,6 +362,16 @@ const resolvers = {
       const response = await clinicalService.uploadClinicalSubmissionData(shortName, filesMap, Authorization);    
       return convertClinicalSubmissionDataToGql(response);
     },
+    validateClinicalSubmissions: async (obj, args, context, info) => {
+      const { Authorization, egoToken } = context;
+      const { shortName, version } = args;
+      // see reason in uploadRegistration
+      if (!TokenUtils.canWriteSomeProgramData(egoToken)) {
```

Comment:
Don't need this, not uploading files, clinical should provide this check as needed (it does)